includes/Modules/Analytics_4.php

```diff
@@ -330,53 +372,77 @@ private function analytics_tracking_opt_out() {
 	 * Provisions new GA4 property and web data stream for provided account.
 	 *
 	 * @since 1.35.0
+	 * @since n.e.x.t Added $account_ticket_params.
 	 *
-	 * @param string $account_id Account ID.
+	 * @param string $account_id            Account ID.
+	 * @param array  $account_ticket_params {
+	 *     Account ticket parameters.
+	 *
+	 *     @type string $accountTicketId Account ticket ID.
+	 *     @type string $propertyName    Property display name.
+	 *     @type string $timezone        Timezone.
+	 *     @type string $profileName     Optional. Profile (View) display name. UA only.
```

Comment:
This is a bit easier to read now, but the inclusion of this UA property here still leads to a sense of cognitive dissonance when reviewing the first line of the description: _Provisions new GA4 property and web data stream for provided account._

By including this (unused) UA property, it makes me question whether this function is actually GA4-only or if it's used for UA too. I then have to read the whole function to make sense of it, as well as tracking back to where it's invoked via the action.

I'd still advocate for removing this property as well as the other unused ones (and avoid passing them in where the action is invoked), to prevent this confusion. Or alternatively, provide a bit of documentation in the description to explain why they are here.

Reply:
Thanks – I've handled this with a new dedicated class as a kind of value object instead which feels a bit cleaner. Let me know what you think.

Reply:
Thanks, that cleans it up nicely. LGTM.